app/javascript/react/components/FruitShowContainer.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from "react"
+import FruitShow from "./FruitShow"
+
+const FruitShowContainer = (props) => {
+  const [fruit, setFruit] = useState({})
+
+  useEffect(() => {
+    const fruitId = props.match.params.id
+
+    fetch(`/api/v1/fruits/${fruitId}`)
+      .then((response) => {
+        if (response.ok) {
+          return response
+        } else {
+          let errorMessage = `${response.status} (${response.statusText})`,
+            error = new Error(errorMessage)
+          throw(error)
+        }
+      })
+      .then(response => response.json())
+      .then(fetchedFruit => {
+        setFruit(fetchedFruit)
+      })
+      .catch(error => console.error(`Error in fetch: ${error.message}`))
+    },[])
+
+
+
+
```

Comment:
I don't know how I missed this last time... you've got a bunch of extra lines here.